events/guildMemberRemove.js

```diff
@@ -18,18 +52,71 @@ module.exports = class {
                 let channel = guild.channels.get(settings.plugins.goodbye.channel);
                 if(channel){
                     let message = settings.plugins.goodbye.message
-                    .replace(/{user}/g, member.user.tag)
+                    .replace(/{user}/g, member)
```

Comment:
I prefer to display the user's nickname without mention because, once the client has restarted, the user will no longer be in the cache (if the client does not have a server in common with the user) and will appear as "<@id>" which is ugly